libtenzir/include/tenzir/plugin.hpp

```diff
@@ -699,7 +699,8 @@ class store_actor_plugin : public virtual plugin {
 /// A base class for plugins that add new store backends.
 class store_plugin : public virtual store_actor_plugin,
                      public virtual parser_parser_plugin,
-                     public virtual printer_parser_plugin {
+                     public virtual printer_parser_plugin,
+                     public virtual printer_serialization_plugin {
```

Comment:
We are missing `parser_serialization_plugin` here.

Reply:
Implemented in https://github.com/tenzir/tenzir/pull/3537/commits/3df56ef98d8221b26dbe8a8d9ef09f48dbf5c487.

`read`ing works now, too.